tcc-transaction-unit-test/src/test/java/org/mengyun/tcctransaction/repository/helper/RedisHelperTest.java

```diff
@@ -0,0 +1,71 @@
+package org.mengyun.tcctransaction.repository.helper;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.powermock.api.mockito.PowerMockito;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+import java.util.Arrays;
+
+public class RedisHelperTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testGetRedisKey() {
+    Assert.assertArrayEquals(new byte[] {(byte)58, (byte)58},
+        RedisHelper.getRedisKey("", ":", ""));
+
+    Assert.assertEquals(Arrays.toString(new Byte[] {65, 71, 71, 103,
+        108, 111, 98, 97, 108, 84, 114, 97, 110, 115, 97, 99, 116,
+            105, 111, 110, 73, 100, 58, 44, 98, 114, 97, 110, 99,
+                104, 81, 117, 97, 108, 105, 102, 105, 101, 114, 58}),
+                    Arrays.toString(RedisHelper.getRedisKey("AGG", "", "")));
+  }
+
+  @Test
+  public void testGetRedisKeyThrowsException() {
+    thrown.expect(NullPointerException.class);
+    RedisHelper.getRedisKey("foo", null);
+  }
+
+  @Test
+  public void testGetVersionKey() {
+    Assert.assertEquals(Arrays.toString(new Byte[]
+        {86, 69, 82, 58, 65, 71, 71, 58}),
+            Arrays.toString(RedisHelper.getVersionKey(
+                "AGG", "", "")));
+  }
+
+  @Test
+  public void testGetVersionKeyThrowsException() {
+    thrown.expect(NullPointerException.class);
+    RedisHelper.getVersionKey("foo", null);
+  }
+
+  @Test
+  public void testExecuteThrowsException() {
+    thrown.expect(NullPointerException.class);
+    RedisHelper.execute(PowerMockito.mock(JedisPool.class), null);
```

Comment:
Also is it not possible to create this without mocking? It looks like it has some constructors but it might not be possible to use them